generator/src/lib.rs

```diff
@@ -2408,7 +2339,7 @@ fn derive_getters_and_setters(
         .map(|extends| {
             let base = name_to_tokens(extends);
             // Extension structs always have a pNext, and therefore always have a lifetime.
-            quote!(unsafe impl Extends<#base<'_>> for #name<'_> {})
+            quote!(unsafe impl<'a> Extends<'a, #base<'a>> for #name<'a> {})
```

Comment:
Named lifetimes are back otherwise the `trybuild` test no longer fails as expected.